README.md

```diff
@@ -21,11 +21,13 @@ It attempts to mimic production setups by making use of OS containers to set up
 Debian testing). If you want to test it in a controlled environment, you can
 use [Vagrant](doc/vagrant.md).
 
-To setup `kube-spawn` on your machine, make sure you have a working [Go
-environment](https://golang.org/doc/install).
+To install `kube-spawn` on your machine, download a single binary release
+or [build from source](#Building).
```

Comment:
s/#Building/#building/
Otherwise the weblink does not work at all, at least with my Firefox browser.